### DISCUSSION TITLE
Problems with Live Forms and Authnet - not sure where to go with this

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1325] [Q&A] [answered]
I have 2 major problems with Live Forms and the "Authoriz.net accept hosted" extension. But not sure if 100% related to Live Forms  or to Vanjaro.

I currently do not have a DNN install to test live forms out.  Where should I post the below information?

1. When I add the extension submissions are not saved when a form is submitted. And do not show up under the Submissions tab
2. After the AuthNet modal form is filled out, it does not close and return the users to the form.

Here is a screencast showing me submitting a form and then looking at the submission. I then added an action to the form to process payment through authnet. Sandbox account accepts payment and then send back to form, but on return I am stuck in the modal.  If I am logged in i get sent to the form in the modal, if not logged in the modal shows the sign in form.
Went back to submissions and the new submission was not recorded.  
Went back to options and removed the authnet payment option.
Submitted another form and the submissions was there.

There is no error in the logs.

I also noticed a notice "Submit a new transaction to PayPal" at the top of the settings form for the extension.

Attached is a screen cast, I cut out the section where I put in the AuthNet keys.

Here is a video displaying the above  https://dev.jcesba.org/share-video

**Comment by ashishpachauriap** [comment, marked as answer]
It seems there is issue with Authorize.NET Accept Hosted extension. Please report this to Mandeeps.com.

**Reply by scottw12** [reply]
Reported at Mandeeps.com
